data/distro.yml

```diff
@@ -487,6 +487,7 @@
         <pre><code>
           <span class="unselectable">$</span> sudo nixos-rebuild switch
         </code></pre>
+        <p>For more details see the <a href="https://nixos.org/manual/nixos/stable/index.html#module-services-flatpak">NixOS documentation</a></p>
```

Comment:
Missing dot at the end of the sentence. Otherwise LGTM.

Reply:
Done in https://github.com/flatpak/flatpak.github.io/pull/475/commits/07762ea466ac3983f4073d0b1303cc7e0b7de056